docs/en/observability/uptime-set-up-heartbeat.asciidoc

```diff
@@ -0,0 +1,130 @@
+[[uptime-set-up-choose-heartbeat]]
+= Use {heartbeat}
+
+{heartbeat} is a lightweight daemon that you install on a remote server to periodically
+check the status of your services and determine if they are available. Performance data is
+then gathered and reported back to {heartbeat}, where it's formatted and sent to the {stack}.
```

Comment:
Sorry for the slightly pedantic suggestion, but just to be precise here I'd say that Heartbeat collects performance data rather than having it reported to it (unless I'm missing something?):

```suggestion
{heartbeat} is a lightweight daemon that you install on a remote server to periodically
check the status of your services and determine if they are available. It gathers performance data,
formats it, and sends the data to the {stack}.
```